### DISCUSSION TITLE
[转贴]中国恐烂尾楼触发金融风暴 拟设房地产基金助开发商缓解债务危机

### DISCUSSION THREAD

**Discussion by TALK1949** [WeNeedHome/SummaryOfLoanSuspension#1015] [General]
原文出处：[中国恐烂尾楼触发金融风暴 拟设房地产基金助开发商缓解债务危机（美国之音）](https://t.co/Fn8B2YACng)

中国当局将设立一个房地产基金以帮助房地产开发商解决严重的债务危机，恢复对房地产业的信心。基金目标最高人民币3000亿元，约440亿美元。这是房地产业债务危机去年爆发以来，当局采取的第一个重大解救步骤。

路透社引述一位熟悉此事的国有银行官员报道，这个基金规模初步定为800亿元，由央行中国人民银行支持。国有的中国建设银行将出资其中的500亿，但资金来自中国人民银行的转融资。如果这个模式有效，其他银行也将跟进，目标是募集2000到3000亿元。

报道说，这位消息人士还表示，这笔资金将用于资助购买未完工的房屋项目并完成建设，然后将房屋出租给个人，作为政府推动出租房屋的一部分。

路透社还引述财经资讯平台REDD的报道说，该基金将支持十多家房地产开发商，包括陷入困境的中国恒大集团。

作为中国经济增长的一个支柱行业，中国当局为遏止房地产业近年的膨胀，采取整肃和整治，导致该行业资金流中断，并从去年开始陷入重大的债务危机，急剧萎缩，拖累了经济增长。

随着房地产债务危机日益加剧，多间房地产开发商面临巨额的债务违约，造成危机不断，更爆发大批建筑项目烂尾，形成烂尾楼风暴，迫使各地已经开始向贷款机构支付房贷的房屋预购者担心开发商无法按期交房而集体停止偿付购房贷款，威胁触发金融风暴，震惊当局，迫使当局紧急采取措施避免房地产业的问题进一步影响更广泛的经济。

据彭博社报道，大批预购房业主拒付房贷有效地促使当局放松对房地产商的放贷。另外一些观察人士表示，政府将建立的基金的规模不足以缓解房地产业的危机。

此外，彭博社还报道，中国当局计划设立该基金的消息传出后，中国房地产股票和美元债券星期一都有所上扬。彭博社有关中国房地产公司的指数上升了1.7%，是一周内的新高。

此次停贷风暴是6月底在江西最先爆发的，那里的烂尾楼业主率先行动，通过网络传播他们的诉求，得到了全国各地处于类似处境的业主们的纷纷响应。

目前，全国已有20多个省市卷入了这次风暴，包括江西、河南、湖南、湖北、廣西、陝西、辽宁、安徽等，风暴所涉楼盘几百个个，贷款机构面临的资金风险高达1.5万亿元人民币，约2200亿美元。

尽管中国监管机构出面要求房地产开发商确保按时向预购者交付房屋，但这种承诺显然并没有打动那些已预购新建房屋的业主，他们表示，将坚持自己的诉求。

**Comment by MrChen2016-08-27** [comment]
资金完成建设用于出租，那租不出去的三四线城市以及以下的，不是没戏了？

**Reply by syyxsxx** [reply]
你太小看老爷们了，公租房捆绑教育和医疗，三四线城市一样租得出去，等房子收够了再慢慢涨房租。

**Reply by TALK1949** [reply]
那也得周边有学校和医院，就像我们这烂尾基本都是郊区的……
况且就算要搞中间不出现吃拿卡要满打满算也才3000亿而已，能救多少都是问题。

**Comment by xujiyou** [comment]
也就恒大的十分之一而已
